execute_jsc.c

```diff
@@ -235,6 +239,11 @@ void proxy_execute_jsc_delayed_init(void) {
     if (!g_proxy_execute_jsc.module)
         return;
 
+    // GObject functions (loaded as a dependency of the module)
```

Comment:
Do we also need to do this in JavaScriptCore one? Load it as a dependency of the module?

Reply:
JavaScriptCore uses `JSGlobalContextRelease()`.